### PR TITLE
Doc updates to openssl_certificate and ecs_certificate

### PR DESCRIPTION
**Pull request by ctrufan**
##### SUMMARY
Some minor fixes to the language and syntax used in the entrust specific certificate modules.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
openssl_certificate
ecs_certificate

### CODE REVIEW

File: lib/ansible/modules/crypto/entrust/ecs_certificate.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 options:
     backup:
         description:
-            - Path to store a backup of the initial certificate, if I(path) pointed to an existing file certificate.
+            - The destination path for a backup of the certificate in I(path).
```
Comment (felixfontein) [inline lib/ansible/modules/crypto/entrust/ecs_certificate.py]:
```suggestion
            - Whether a backup should be made for the certificate in I(path).
```
After all, it's a boolean option, and not the return value (which will contain the path) :)

**Review by felixfontein — CHANGES_REQUESTED** [review]
Besides the one option, looks good!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@ctrufan thanks for caring about the docs! :)

Do you want to create a backport PR for `stable-2.9`?

**Comment by ctrufan** [comment]
I wasn't sure if it was too late, I'll create a backport tomorrow!

**Comment by felixfontein** [comment]
It could very well be too late for 2.9.0, but should definitely make it for 2.9.1. And docs changes are sometimes also merged between releases, so it could show up even earlier than 2.9.1.

**Comment by ctrufan** [comment]
@felixfontein 
Thanks for the review and merge!

**Pull request merged**